hack/lib/test.sh

```diff
@@ -246,3 +246,78 @@ kube::test::if_supports_resource() {
   done
   return 1
 }
+
+
+kube::test::version::object_to_file() {
```

Comment:
the function isnt a test, its a helper function. I suppose i can rename it to make that clear.